### DISCUSSION TITLE
Error PVGeo: module 'PVGeo' has no attribute 'filters'

### DISCUSSION THREAD

**Discussion by TTESimone** [pyvista/pyvista#5739] [Q&A]
Hi,

I tried:
grid1 = PVGeo.filters.VoxelizePoints().apply(cloud_fase1)

But get the error module 'PVGeo' has no attribute 'filters'
On the page: https://pvgeo.org/examples/filters-general/voxelize-points.html it is described to use it this way.

What is going wrong?
I use PVGe 3.0.1

**Comment by banesullivan** [comment]
Sounds like you have a folder named pvgeo in your working directory that is conflicting with the import
